module/support/logging/DataLogging/data/config/webotsrobocup/DataLogging.yaml

```
@@ -1,7 +1,7 @@
 output:
   directory: log
   split_size: 5 * GiB
-
+  max_size: 10630044057
```

magic. Needs a reason/context
```suggestion
  max_size: 10630044057
```